### ISSUE TITLE
PCAWG does not show the usual charts first on Study View by default

### ISSUE THREAD

**Issue by inodb** [cBioPortal/cbioportal#10952]
Expecting to see cancer type / cancer type detailed:

<img width="729" alt="image" src="https://github.com/user-attachments/assets/3eac7e71-9e35-4f2a-8e41-cdbd298aab8b">

**Comment by alisman** [comment]
@inodb i do see the charts by default.  DO you mean see them at the abs top?

![Image](https://github.com/user-attachments/assets/526c73ff-a57c-4f45-b250-250a571d10d1)

**Comment by inodb** [comment]
@alisman yea exactly - all other studues have the cancer type as the top left chart afaik (which makes it easy to find):

see e.g. https://www.cbioportal.org/study/summary?id=msk_impact_2017
![Image](https://github.com/user-attachments/assets/29ec846b-24d8-4015-a351-9941053a3135)

**Comment by tmazor** [comment]
I wonder if the priority is set for these charts in the data file? I think that will override the default priorities which normally put those charts first.

**Comment by alisman** [comment]
@inodb reopining.  i think i found them problem.
@ritikakundra @sbabyanusha 
Pretty sure this is a data issue.  The cancer_type and cancer_type_detailed attributes have priority=3 whereas for studies which are showing properly, they have priority=1.  Can we change them to 1?
![image](https://github.com/user-attachments/assets/e2e0c50b-2458-48bb-bdc7-dd4790f1fe75)

**Comment by inodb** [comment]
Thanks for digging into this @alisman ! 

@rmadupuri do you know if the priority of the clinical attributes were added on purpose for PCAWG? (See PR: https://github.com/cBioPortal/datahub/pull/1534). Thanks!

**Comment by rmadupuri** [comment]
@inodb yes, a couple attributes were set a higher priority on purpose but ideally the cancer type and cancer type detailed charts should appear before the dynamically generated charts. We will test and fix this in data.

**Comment by rmadupuri** [comment]
In the data files `Cancer Type` and `Cancer Type Detailed` were given the highest priority of 3. Is there any logic on the frontend that causes these charts to appear after the default ones if their priority is not exactly set to 1?

**Comment by alisman** [comment]
@rmadupuri that's a good question.  i was thinking 1 was HIGHEST priority.  let me look

**Comment by rmadupuri** [comment]
Thanks for sharing the documentation @alisman, it makes sense now. I will go ahead and reset the priority for both the cancer type and cancer type charts to 1.

**Comment by rmadupuri** [comment]
@alisman @inodb this is now fixed in the portal. Thanks for catching and figuring it out!

![Image](https://github.com/user-attachments/assets/db141ca1-c402-485f-a042-3a44cdb7db7f)

**Comment by inodb** [comment]
Awesome - thanks @rmadupuri !
